searx/static/themes/simple/.eslintrc.json

```diff
@@ -1,28 +0,0 @@
-{
-    "env": {
-        "browser": true,
-        "es2021": true,
-        "node": true
-    },
```

Comment:
In ESlint 9.0 there is no longer `env` ... I also don't know what these values were needed for and whether they are still needed.